en/ability.json

```
@@ -305,103 +305,103 @@
+    "description": "Moves targetting the Pokémon have their accuracy halved while the Pokémon is confused."
   },
   "motorDrive": {
     "name": "Motor Drive",
-    "description": "Boosts its Speed stat if hit by an Electric-type move instead of taking damage."
+    "description": "The Pokémon takes no damage when hit by Electric-type moves. Instead, its Speed stat is boosted."
   },
   "rivalry": {
     "name": "Rivalry",
-    "description": "Becomes competitive and deals more damage to Pokémon of the same gender, but deals less to Pokémon of the opposite gender."
+    "description": "The Pokémon’s competitive spirit makes it deal 25% more damage to Pokémon of the same gender, but 25% less damage to Pokémon of the opposite gender."
   },
   "steadfast": {
     "name": "Steadfast",
-    "description": "The Pokémon’s determination boosts the Speed stat each time the Pokémon flinches."
+    "description": "The Pokémon’s determination boosts its Speed stat every time it flinches."
   },
   "snowCloak": {
     "name": "Snow Cloak",
-    "description": "Boosts the Pokémon’s evasiveness in snow."
+    "description": "Boosts the Pokémon’s evasiveness in snow or hail by 20%. It also takes no damage from hail."
   },
   "gluttony": {
     "name": "Gluttony",
-    "description": "Makes the Pokémon eat a held Berry when its HP drops to half or less, which is sooner than usual."
+    "description": "If the Pokémon is holding a Berry to be eaten when its HP is 1/4 or less such as a Starf Berry, it will instead eat the Berry when its HP drops to 1/2 or less."
   },
   "angerPoint": {
     "name": "Anger Point",
-    "description": "The Pokémon is angered when it takes a critical hit, maxing its Attack stat."
+    "description": "The Pokémon is angered when it takes a critical hit, and that maxes its Attack stat."
   },
   "unburden": {
     "name": "Unburden",
-    "description": "Boosts the Speed stat if the Pokémon’s held item is used or lost."
+    "description": "Doubles the Speed stat of the Pokémon if its held item is used or lost."
   },
   "heatproof": {
     "name": "Heatproof",
-    "description": "The heatproof body of the Pokémon halves the damage from Fire-type moves that hit it."
+    "description": "The Pokémon’s heatproof body halves the damage taken from Fire-type moves and being burned."
   },
   "simple": {
     "name": "Simple",
-    "description": "The stat changes the Pokémon receives are doubled."
+    "description": "Doubles the effects of the Pokémon’s stat changes."
   },
   "drySkin": {
     "name": "Dry Skin",
-    "description": "Restores HP in rain or when hit by Water-type moves. Reduces HP in harsh sunlight, and increases the damage received from Fire-type moves."
+    "description": "Restores the Pokémon’s HP in rain by 1/8 of its max HP or 1/4 when it is hit by Water-type moves. Reduces its HP by 1/8 of its max HP in harsh sunlight, and receives 25% more damage from Fire-type moves."
```

```suggestion
    "description": "Restores or reduces the Pokémon’s HP by 1/8 of its max HP in rain and harsh sunlight, respectively. The Pokémon also restores its HP by up to 1/4 of its max HP when hit by Water-type moves, but receives 25% more damage from Fire-type moves."
```
Kind of messy with all the values so might need restructuring like this